Lib/test/test_capi/test_long.py

```diff
@@ -211,9 +211,10 @@ def check_long_asintandoverflow(self, func, min_val, max_val):
 
         self.assertEqual(func(min_val - 1), (-1, -1))
         self.assertEqual(func(max_val + 1), (-1, +1))
+        with self.assertRaises(SystemError):
+            func(None)
```

Comment:
it seems this form more preferred in the file:
```suggestion
        self.assertRaises(SystemError, func, None)
```